extensions/amp-story/1.0/amp-story-interactive-poll.js

```diff
@@ -201,7 +205,30 @@ export class AmpStoryInteractivePoll extends AmpStoryInteractive {
       currOption.querySelector(
         '.i-amphtml-story-interactive-option-percentage-text'
       ).textContent = `${percentage}%`;
-      currOption.setAttribute('style', `flex-grow: ${percentage} !important`);
+
+      currOption
+        .querySelector('.i-amphtml-story-interactive-option-percent-bar')
+        .setAttribute(
+          'style',
+          `transform: scaleX(${percentage * 0.01 * 2}) !important`
+        );
```

Comment:
Using setStyle is usually a better idea than setting the style attribute manually, as it automatically checks if there's a style applied already and appends this property

Reply:
This works in most cases.
I need to set this manually for `.i-amphtml-story-interactive-option-text-container` so that I can append `!important`.